### PR TITLE
Add lmodel namespace

### PR DESCRIPTION
**Pull request by noelmcloughlin**
This PR adds lmodel/ namespace for https://github.com/lmodel

### CODE REVIEW

**Comment by davidlehn** [comment]
Have you considered using only one top level `.htaccess` file?  It's usually easier to maintain than a whole tree of files with single rules.

File: lmodel/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# /lmodel/
+
+Perma-id service for https://github.com/lmodel
+
+## Maintainer
+
+Noel McLoughlin (noel.mcloughlin AT gmail.com)
```
Comment (davidlehn) [inline lmodel/README.md:7]: Can you please add your github account id here too?

**Comment by noelmcloughlin** [comment]
Thanks @davidlehn  for suggestions, done

**Comment by dgarijo** [comment]
@noelmcloughlin I added a small comment. Please have a look

File: lmodel/.htaccess [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/ld+json .jsonld
+AddType application/vnd.ms-excel .xlsx
+AddType application/json .json
+AddType text/yaml .yaml .yml
+AddType application/json+shexschema .shex
+AddType application/octet-stream .sql .proto
```
[the remark this answers is not in the document]
Comment (noelmcloughlin) [reply lmodel/.htaccess:14]: I don't know much about htaccess - I will remove them anyway.

**Pull request merged**